test/helpers/contract-helpers.ts

```diff
@@ -1,38 +1,72 @@
 // Imports
 declare const ethers: any;
-
+import { SSVKeys, KeyShares, EncryptShare } from 'ssv-keys';
 import { trackGas, GasGroup } from './gas-usage';
+import { Validator, Operator } from './types';
+import validatorKeys from './json/validatorKeys.json';
+import operatorKeys from './json/operatorKeys.json';
 
 export let DB: any;
 export let CONFIG: any;
 export let SSV_MODULES: any;
 
-const SIGNATURE = "869ff0ade8cb76048fb3aa8d4c26b0d71f96eb3bfb69144c080e7fe8db6e1e2f926f365886a59a25601b290f5f4695dd5b7e8cce1c462bdef5bebee34d94711541baeadbc7546a8c3ca301d86aaab4767c809a"
+export const DEFAULT_OPERATOR_IDS = {
```

Comment:
@Mohsen-T do we have an alternative to create these default lists of operators? I remember you created some helper functions when integrating with ssv-keys. thanks!

Reply:
In the ssv-keys integration branch, I have not developed specific helper functions for generating default operator IDs. 
Therefore, at that time, I've utilized constant arrays, specifically [1, 2, 3, 4], to define these values.